test/distributed/org/apache/cassandra/distributed/test/SimpleReadWriteTest.java

```diff
@@ -23,20 +23,16 @@
 import org.junit.Assert;
 import org.junit.Test;
 
-import org.apache.cassandra.config.DatabaseDescriptor;
 import org.apache.cassandra.db.Keyspace;
 import org.apache.cassandra.distributed.Cluster;
 import org.apache.cassandra.distributed.api.ConsistencyLevel;
 import org.apache.cassandra.distributed.api.ICluster;
 import org.apache.cassandra.distributed.api.IInvokableInstance;
-import org.apache.cassandra.metrics.ReadRepairMetrics;
 import org.apache.cassandra.io.sstable.format.SSTableReader;
 
 import static org.apache.cassandra.distributed.api.Feature.NETWORK;
 import static org.apache.cassandra.distributed.shared.AssertUtils.*;
 import static org.apache.cassandra.net.OutboundConnections.LARGE_MESSAGE_THRESHOLD;
-import static org.apache.cassandra.net.Verb.READ_REPAIR_REQ;
-import static org.apache.cassandra.net.Verb.READ_REPAIR_RSP;
 import static org.junit.Assert.fail;
 
 // TODO: this test should be removed after running in-jvm dtests is set up via the shared API repository
```

Comment:
@adelapena Let's change `ICluster` -> `ICluster<?>` to get rid of all the compiler warnings :)